hash/dc.xml

```diff
@@ -489,7 +489,7 @@ P830-****-** : Appears to be a Sega part number for promo discs.
 		<rom name="4 Wheel Thunder (Europe) (En,Fr) (Track 17).bin" size="42691152" crc="98d76c3a" md5="70a968acd12e3fbc6aceb342313df386" sha1="1a25c39dd895500fa5a5e167d760f95bab46a4c8"/>
 		<rom name="4 Wheel Thunder (Europe) (En,Fr) (Track 18).bin" size="339076080" crc="628fd0fa" md5="af00c0bb0afb9e8027382e0e051b9893" sha1="c7d7ea0bbe6b306cdd4757f9519d3b96282ce768"/>
 		-->
-		<description>4 Wheel Thunder (Europe)</description>
+		<description>4 Wheel Thunder (Europe) (En,Fr)</description>
```

Comment:
The en,fr should be translated into `<info name="language" value="English/French" />` and the description tag reverted.

Reply:
The purpose of the languages in the title was to match the redump title exactly, if we were shifting these to a language tag then another tag would be required to match the redump source, based on some of the other suggestions/objections that may be the better way to go.